_data/examples.json

```diff
@@ -233,20 +264,45 @@
       "title": "Error Bars showing Standard Deviation"
     },
     {
-      "name": "layer_histogram_global_mean",
-      "title": "Histogram with a Global Mean Overlay"
+      "name": "layer_area_ci",
+      "title": "Line chart with Confidence Interval Band"
     },
     {
       "name": "layer_global_mean_dev",
       "title": "Scatterplot with Mean and Standard Deviation Overlay"
+    }
+  ],
+  "Box Plots": [
+    {
+      "name": "box-plot_minmax_2D_vertical_normalized",
+      "title": "Box Plot with Min/Max Whiskers"
     },
     {
-      "name": "layer_area_ci",
-      "title": "Line chart with Confidence Interval Band"
+      "name": "box-plot_tukey_2D_vertical_normalized",
+      "title": "Tukey Box Plot (1.5 IQR)"
+    }
+  ],
+  "Other Layered Charts": [
+    {
+      "name": "layer_bar_annotations",
+      "title": "Bar chart that highlights values beyond a threshold",
+      "description": "To create a bar chart that highlights values beyond a threshold, we use two `layer`s of `bar` marks. The lower layer shows all the bars while the upper layer shows bar with values above the threshold in red (`#e45755`).  We then `layer` a `rule` mark and a `text` mark over the bars to annotate the threshold value."
+    },
+    {
+      "name": "layer_precipitation_mean",
+      "title": "Mean overlay over precipitation chart"
+    },
+    {
+      "name": "layer_candlestick",
+      "title": "Candlestick Chart"
+    },
+    {
+      "name": "layer_histogram_global_mean",
+      "title": "Histogram with a Global Mean Overlay"
     },
     {
       "name": "layer_falkensee",
-      "title": "Falkensee",
+      "title": "Line chart with Hilighted Rectangles",
```

Comment:
typo

Reply:
chart => Chart? 

(Falkensee is a terrible name as it doesn't suggest what this example is for at all.

Reply:
Hilighted

Reply:
> Falkensee is a terrible name

Also, thanks.